VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -116,9 +134,148 @@ export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implemen
             });
     }
 
+    private initialiseVideoControlCacheLogic() {
+        this.hearingCountdownFinishedSubscription = this.eventService.getHearingCountdownCompleteMessage().subscribe(() => {
+            this.conferenceStatusChangedSubscription?.unsubscribe();
+            this.conferenceStatusChangedSubscription = this.conferenceService.onCurrentConferenceStatusChanged$.subscribe(
+                conferenceStatus => this.onConferenceStatusChanged(conferenceStatus)
+            );
+            this.onParticipantOrVmrPexipConnectedOrIdUpdatedSubscription?.unsubscribe();
+            this.onParticipantOrVmrPexipConnectedOrIdUpdatedSubscription = merge<ParticipantModel | VirtualMeetingRoomModel>(
+                this.participantService.onParticipantConnectedToPexip$,
+                this.participantService.onParticipantPexipIdChanged$,
+                this.participantService.onVmrConnectedToPexip$,
+                this.participantService.onVmrPexipIdChanged$
+            ).subscribe(() => this.restoreSpotlightState());
+        });
+
+        this.participantStatusChangedSubscription = this.participantService.onParticipantStatusChanged$.subscribe(participant =>
+            this.updateSpotlightStateOnParticipantDisconnectDuringConference(participant)
+        );
+
+        this.onConferenceStatusChangedSubscription = this.conferenceService.onCurrentConferenceStatusChanged$.subscribe(update =>
+            this.onConferenceInSessionCheckForDisconnectedParticipants(update)
+        );
+    }
+
+    onConferenceStatusChanged(conferenceStatus: { oldStatus: ConferenceStatus; newStatus: ConferenceStatus }): void {
+        if (conferenceStatus.newStatus === ConferenceStatus.InSession) {
+            this.logger.info(`${this.loggerPrefixJudge} spotlighting judge as it is the start of the hearing`);
+
+            let participants = this.participantService.participants;
+
+            if (conferenceStatus.oldStatus === ConferenceStatus.NotStarted) {
```

Comment:
Why do we only do this when the old status was NotStarted?

Reply:
Oh, we have a different status for when it's Paused and in that case we don't set the judge as spotlight.